com.stepmania.StepMania.json

```diff
@@ -0,0 +1,54 @@
+{
+    "app-id" : "com.stepmania.StepMania",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "stepmania",
+    "finish-args" : [
+        "--device=all",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--socket=x11",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules" : [
+        "libmad/libmad.json",
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/glew/glew.json",
+        "shared-modules/udev/udev-175.json",
+        {
+            "name" : "stepmania",
+            "buildsystem" : "cmake",
+            "config-opts" : [
+                "-DWITH_FFMPEG:BOOL=OFF"
+            ],
+            "post-install": [
+                "mkdir -p /app/bin/",
+                "ln -s /app/stepmania-5.1/stepmania /app/bin/stepmania",
+                "mkdir -p /app/share/applications/",
+                "install -m644 -p stepmania.desktop /app/share/applications/com.stepmania.StepMania.desktop",
+                "mkdir -p /app/share/icons/hicolor/256x256/apps/",
+                "install -m644 -p icons/hicolor/256x256/apps/stepmania-ssc.png /app/share/icons/hicolor/256x256/apps/com.stepmania.StepMania.png",
+                "mkdir -p /app/share/icons/hicolor/scalable/apps/",
+                "install -m644 -p icons/hicolor/scalable/apps/stepmania-ssc.svg /app/share/icons/hicolor/scalable/apps/com.stepmania.StepMania.svg"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/stepmania/stepmania/archive/v5.1.0-b1.tar.gz",
+                    "sha256" : "97067d72481407635c92ea4c1806b998b8354250d9c581c048e62e3461757012"
+                },
+                {
+                    "type": "patch",
+                    "path": "stepmania-desktop.diff"
```

Comment:
Just use `desktop-file-edit` in a post-install step instead of maintaining a patch.